hack/update_codegen.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
"hack" because every  custom operator implementation I looked at, that uses these regeneration scripts used this naming convention. I found it a  bit strange but went with the common naming. No problem to rename.

We spoke about this at the beginning. I initially tried not to have the generated code in the repo and generate it during CI, but that  did not work. So I committed the generated code and left improvements for later.

Reply:
I dont seem to be able to comment on your comment regarding the flags. Yes, as I mentioned to you a couple of days ago, that part needs tidying up. Too early for a PR ....

Reply:
> I initially tried not to have the generated code in the repo and generate it during CI, but that did not work.

How did it fail to work?